pyproject.toml

```diff
@@ -59,8 +59,12 @@ smolagents = [
   "smolagents[mcp]>=1.14.0",
 ]
 
+serve = [
+  "a2a-sdk",
+]
+
 all = [
-  "any-agent[agno,google,langchain,llama_index,openai,smolagents]",
+  "any-agent[agno,google,langchain,llama_index,openai,smolagents,serve]",
```

Comment:
I would not put this as part of `all` and instead clarify in the docs how to install for serving.

I would like for us to stop advertising `all` in the docs and instead default to use what works with the minimal built-in deps (tinyagent)

Reply:
https://github.com/mozilla-ai/any-agent/issues/317